scripts/shared/lib/deploy_helm

```diff
@@ -24,7 +24,8 @@ function install_helm() {
 
 function deploytool_prereqs() {
     helm init --client-only
-    helm repo add submariner-latest https://submariner-io.github.io/submariner-charts/charts
+    #helm repo add submariner-latest https://submariner-io.github.io/submariner-charts/charts
+    helm repo add submariner-latest http://172.17.0.2:8080 
```

Comment:
This shouldn’t be part of the commit ;-).

Reply:
done.